cmake/projects/odb-compiler/hunter.cmake

```diff
@@ -29,9 +29,9 @@ elseif(CMAKE_HOST_UNIX)
       VERSION
       "2.4.0"
       URL
-      "http://www.codesynthesis.com/download/odb/2.4/odb-2.4.0-i686-linux-gnu.tar.bz2"
```

Comment:
> I guess this would be the responsibility of the toolchain, no?

Yes. As far as I understand value of `CMAKE_SIZEOF_VOID_P` will depend on toolchain you're using since it determined by `try_compile`. Or you mean we need host architecture?